public/index.html

```diff
@@ -53,6 +53,10 @@
         );
       }
     </script>
+    <script
+      src="https://cdn.ravenjs.com/3.19.1/raven.min.js"
```

Comment:
note that this introduce tracking by ravenjs.com which may be considered as a bad practice.